openssl/thread_setup.c

```diff
@@ -0,0 +1,46 @@
+//go:build linux && !android
+// +build linux,!android
+
+#include "goopenssl.h"
+
+#include <stdio.h>
+#include <pthread.h>
+#include <sys/types.h>
+#include <sys/syscall.h>
+
+#define _GNU_SOURCE
+#include <unistd.h>
+ 
+#define CRYPTO_LOCK      0x01
```

Comment:
This used to be aligned with some adjacent lines, but they aren't here anymore:

```suggestion
#define CRYPTO_LOCK 0x01
```